src/types/video.ts

```diff
@@ -274,6 +274,16 @@ export type ControlsStyles = {
   liveLabel?: string;
 };
 
+export declare enum ContextualAction {
+  SKIP_INTRO = 'SkipIntro',
+  NEXT_EPISODE = 'NextEpisode',
+}
+export type ContextualActions = {
+  action: ContextualAction;
+  startAt?: number;
```

Comment:
And I also wondering if: endAt is the good api ? what will happen if endAt < startAt. A duration would be easier to use ? (This is minor and not blocking the merge)